elated-egrets/bangazon-internal-site
# Pzc department url

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git checkout pzc-department-url
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. delete db/migrations
1. remake migrations
1. fill DB with fake data
1. pyman runserver
1. navigate to department/
1. See list of departments printed on page

## What you changed
Added view and template for department page


## What was added to read me
Nothing

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
